pom.xml

```diff
@@ -4,32 +4,206 @@
+        <jenkins.version>2.60.3</jenkins.version>
+        <java.level>8</java.level>
+        <commons-lang3.version>3.8.1</commons-lang3.version>
+        <!-- log4j-specific properties -->
+        <log4j.version>2.11.1</log4j.version>
+        <log4j-audit.version>1.0.0</log4j-audit.version>
+        <!-- maven enforcer plugin properties -->
+        <maven-enforcer-plugin.version>3.0.0-M2</maven-enforcer-plugin.version>
+        <maven.enforcer.requireMavenVersion>[3.6.0,)</maven.enforcer.requireMavenVersion>
+        <extra-enforcer-rules.version>1.0-beta-9</extra-enforcer-rules.version>
+        <maven.enforcer.requireJavaVersion>8</maven.enforcer.requireJavaVersion>
+        <maven.enforcer.maxJdkVersion>1.8</maven.enforcer.maxJdkVersion>
+        <!-- other maven-specific properties -->
+        <maven-hpi-plugin.version>2.7</maven-hpi-plugin.version>
+        <maven.compiler.source>[1.8,)</maven.compiler.source>
+        <maven.compiler.target>[1.8,)</maven.compiler.target>
+        <versions-maven-plugin.version>2.7</versions-maven-plugin.version>
+        <!-- <maven.compiler.release>9</maven.compiler.release> -->
+    </properties>
+
+    <!-- Assuming you want to host on @jenkinsci: <scm> <connection>scm:git:git://github.com/jenkinsci/${project.artifactId}-plugin.git</connection>
+    <developerConnection>scm:git:git@github.com:jenkinsci/${project.artifactId}-plugin.git</developerConnection> <url>https://github.com/jenkinsci/${project.artifactId}-plugin</url> </scm> -->
+
+    <!-- defines all project dependencies -->
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-audit-api</artifactId>
+            <version>${log4j-audit.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
+            <version>${commons-lang3.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-api</artifactId>
+            <version>${log4j.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-core</artifactId>
+            <version>${log4j.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-catalog-api</artifactId>
+            <version>${log4j-audit.version}</version>
+        </dependency>
+    </dependencies>
+
+    <!-- handle the maven build lifecycle phases -->
+    <build>
+        <plugins>
+            <!-- handles creation of jenkins plugin packages -->
+            <plugin>
+                <groupId>org.jenkins-ci.tools</groupId>
+                <artifactId>maven-hpi-plugin</artifactId>
+                <version>${maven-hpi-plugin.version}</version>
```

Comment:
This version doesn't need to be defined here because it's already specified by the parent pom. You can simply remove the entire version element here.